### DISCUSSION TITLE
How long will PHP 7.4 on Ubuntu 18.04 be available?

### DISCUSSION THREAD

**Discussion by mfn** [oerdnj/deb.sury.org#1740] [Q&A] [answered]
Apologies if this was mentioned somewhere else and I didn't find it, I was wondering what the current EOL for PHP 7.4 on 18.04 would be.

Piecing together information I found:
- PHP on Ubuntu 18.04 would be available as long as 18.04 is LTS, [which would mean 2023](https://wiki.ubuntu.com/Releases)
- However [PHP 7.4 EOL is this year in November](https://www.php.net/supported-versions.php)

Is it correct to assume that availability of your packages of PHP 7.4 for Ubuntu 18.04 would cease to exist after Nov 2022?

Thank you,
- Markus

**Comment by oerdnj** [comment, marked as answer]
The README of this repository says:

> This GitHub project is the issue tracker of PPA for PHP (5.6, 7.0, 7.1, 7.2, 7.3, 7.4, 8.0 and 8.1).

Now, would it make sense to stop supporting PHP 7.4 and continue supporting PHP 5.6?

**Comment by mfn** [comment]
Fair point, guess I looked at all the wrong places; thank you 🙏🏼

**Reply by oerdnj** [reply]
I guess I (or someone) could add a section to the FAQ on the topic, but I thought it's generally well understood. Also see my posts on the topic here: https://www.patreon.com/oerdnj
